### ISSUE TITLE
Recreating paper results

### ISSUE THREAD

**Issue by tomk2005** [arcchang1236/CA-NoiseGAN#6]
Hello,

I am having difficulty recreating your paper results.
I'm currently running your code on the data you provided with pre-trained models with the config values of:
patch_size = [64, 64]
sample_amount = 1191

I then average KLD and get KLD ~0.5 .
Should running test noise models.py with these settings recreate the paper's KLD (~0.00159)?

If so, can you possibly upload a more detailed requirements files of your environment? maybe it is some package mismatch?

**Comment by yishionsunshine** [comment]
I have the same problem as user tomk2005, can you upload more detailed requirements?

**Comment by OrGreenberg** [comment]
Hi!
same for me. 

Here are the results I achieved running the code over the provided test set, without any changes in the config file, except for the data_dir. I used the pretrained models provided.

--> Denoiser Test (PSNR)

average psnr ours: 47.841578781906506
average psnr g: 41.00051686254721
average psnr pg: 46.08441194932963
average psnr noiseflow: 47.147988764349506
average psnr ours real: 47.71135122092602
average psnr real: 46.7334418265441

--> Noise Models Test (KLD): 

G: 0.6178 | PG: 0.0685 | Ours: 0.1171  [** noise-flow results are not presented]
The denoiser's results make sense, yet not fully aligned with the results presented in the paper (Table 7).
On the other hand, the noise model results are completely

**Comment by happycaoyue** [comment]
> Hi! same for me.
> 
> Here are the results I achieved running the code over the provided test set, without any changes in the config file, except for the data_dir. I used the pretrained models provided.
> 
> --> Denoiser Test (PSNR)
> 
> average psnr ours: 47.841578781906506 average psnr g: 41.00051686254721 average psnr pg: 46.08441194932963 average psnr noiseflow: 47.147988764349506 average psnr ours real: 47.71135122092602 average psnr real: 46.7334418265441
> 
> --> Noise Models Test (KLD):
> 
> G: 0.6996052614889056 | PG: 0.06394353525639518 | Ours (CA-NoiseGAN): 0.03184607044417602 [** noise-flow results are not presented]
> 
> The denoiser's results make sense, yet not fully aligned with the results presented in the paper (Table 7). On the other hand, the noise model results seem completely wrong, particularly compared to the ones presented in the paper (Table 1).
> 
> Any help here? :)

Can you leave an email address for me? Let's talk about noise modelling?
